src/components/Chips/icons/Chevron.tsx

```diff
@@ -0,0 +1,14 @@
+import SvgIcon, { SvgIconProps } from '@mui/material/SvgIcon';
+
+export const Chevron = (props: SvgIconProps) => {
+  return (
+    <SvgIcon {...props} width="20" height="20" viewBox="0 0 20 20" fill="none">
+      <path
+        fillRule="evenodd"
+        clipRule="evenodd"
+        d="M9.46973 12.5304L5.46973 8.53039L6.53039 7.46973L10.0001 10.9394L13.4697 7.46973L14.5304 8.53039L10.5304 12.5304L10.0001 13.0607L9.46973 12.5304Z"
+        fill="rgba(0, 0, 0, 0.34)"
```

Comment:
Ага, ты еще с этим не сталкивался =)

```suggestion
        fill="currentColor"
```
Это нужно чтобы перекрашивать иконку через css с помощью свойства color